easybuild/easyconfigs/p/PLINK/PLINK-2.00-alpha2-x86_64.eb

```diff
@@ -6,16 +6,12 @@ version = '2.00-alpha2-x86_64'
 homepage = 'https://www.cog-genomics.org/plink/2.0/'
 description = 'Whole-genome association analysis toolset'
 
-toolchain = {'name': 'dummy', 'version': 'dummy'}
+toolchain = SYSTEM
 
 source_urls = ['https://s3.amazonaws.com/plink2-assets/']
 sources = ['plink2_linux_x86_64_20190810.zip']
 checksums = ['009e9ae14aa821485b344846bfa46a8886e48f17b8feaa23c6a0360f61ea445f']
 
-files_to_copy = [
```

Comment:
why this?

Reply:
`PLINK` is installed with `PackedBinary`, while `files_to_copy` is custom to `MakeCp`, so it's not actually used here at all...

Reply:
It must be removed here for the easyconfig tests to pass, since they now fail if local variables are used that don't follow the recommended naming scheme, see https://easybuild.readthedocs.io/en/4.x/Easyconfig-files-local-variables.html